src/listbox/Listbox.ts

```diff
@@ -0,0 +1,264 @@
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { DNode } from '@dojo/widget-core/interfaces';
+import { ThemeableMixin, ThemeableProperties, theme } from '@dojo/widget-core/mixins/Themeable';
+import { diffProperty } from '@dojo/widget-core/decorators/diffProperty';
+import { auto, reference } from '@dojo/widget-core/diff';
+import { v, w } from '@dojo/widget-core/d';
+import Dimensions from '@dojo/widget-core/meta/Dimensions';
+import uuid from '@dojo/core/uuid';
+import { Keys } from '../common/util';
+import * as css from './styles/listbox.m.css';
+
+/* Listbox Option sub-widget */
+export interface ListboxOptionProperties extends ThemeableProperties {
```

Comment:
Good question 😂 . It used to be, then after talking with Ant/Matt G, the conclusion was that sub-widgets shouldn't really exist because they don't stand on their own, except that they make handling certain events much easier. I moved it out of its own file so it would seem less separate, but I agree that it seemed better organized when separate

Reply:
Moved back to its own file. We'll see what happens 😄